packages/vats/test/bootstrapTests/test-liquidation.js

```diff
@@ -0,0 +1,592 @@
+      },
+      MintFee: {
+        type: 'ratio',
+        value: { numerator: { value: 50n }, denominator: { value: 10_000n } },
+      },
+    },
+  });
+  t.like(readLatest('published.auction.governance'), {
+    current: {
+      AuctionStartDelay: { type: 'relativeTime', value: { relValue: 2n } },
+      ClockStep: {
+        type: 'relativeTime',
+        value: { relValue: 3n * SECONDS_PER_MINUTE },
+      },
+      DiscountStep: { type: 'nat', value: 500n }, // 5%
+      LowestRate: { type: 'nat', value: 6500n }, // 65%
+      PriceLockPeriod: {
+        type: 'relativeTime',
+        value: { relValue: SECONDS_PER_HOUR / 2n },
+      },
+      StartFrequency: {
+        type: 'relativeTime',
+        value: { relValue: SECONDS_PER_HOUR },
+      },
+      StartingRate: { type: 'nat', value: 10500n }, // 105%
+    },
+  });
+
+  const borrower = await walletFactoryDriver.provideSmartWallet(
+    'agoric1borrower',
+  );
+
+  await borrower.executeOfferMaker(Offers.vaults.OpenVault, {
+    offerId: 'open-vault1',
+    collateralBrandKey,
+    wantMinted: setup.vaults[0].ist,
+    giveCollateral: setup.vaults[0].atom,
+  });
+  t.like(borrower.getLatestUpdateRecord(), {
+    updated: 'offerStatus',
+    status: { id: 'open-vault1', numWantsSatisfied: 1 },
+  });
+
+  await borrower.executeOfferMaker(Offers.vaults.OpenVault, {
+    offerId: 'open-vault2',
+    collateralBrandKey,
+    wantMinted: setup.vaults[1].ist,
+    giveCollateral: setup.vaults[1].atom,
+  });
+  t.like(borrower.getLatestUpdateRecord(), {
+    updated: 'offerStatus',
+    status: { id: 'open-vault2', numWantsSatisfied: 1 },
+  });
+
+  await borrower.executeOfferMaker(Offers.vaults.OpenVault, {
+    offerId: 'open-vault3',
+    collateralBrandKey,
+    wantMinted: setup.vaults[2].ist,
+    giveCollateral: setup.vaults[2].atom,
+  });
+  t.like(borrower.getLatestUpdateRecord(), {
+    updated: 'offerStatus',
+    status: { id: 'open-vault3', numWantsSatisfied: 1 },
+  });
+
+  // Verify starting balances
+  t.like(readLatest('published.vaultFactory.managers.manager0.vaults.vault0'), {
+    debtSnapshot: { debt: { value: scale6(setup.vaults[0].debt) } },
+    locked: { value: scale6(setup.vaults[0].atom) },
+    vaultState: 'active',
+  });
+  t.like(readLatest('published.vaultFactory.managers.manager0.vaults.vault1'), {
+    debtSnapshot: { debt: { value: scale6(setup.vaults[1].debt) } },
+    locked: { value: scale6(setup.vaults[1].atom) },
+    vaultState: 'active',
+  });
+  t.like(readLatest('published.vaultFactory.managers.manager0.vaults.vault2'), {
+    debtSnapshot: { debt: { value: scale6(setup.vaults[2].debt) } },
+    locked: { value: scale6(setup.vaults[2].atom) },
+    vaultState: 'active',
+  });
+
+  const buyer = await walletFactoryDriver.provideSmartWallet('agoric1buyer');
+  {
+    // ---------------
+    //  Place bids
+    //   Bid 1: 80 IST at 10% discount
+    //   Bid 2: 90 IST at price of $9
+    //   Bid 3: 150 IST at 15% discount
```

Comment:
this duplicates setup and getting out of sync won't be automatically detected. prune?

Reply:
good point, done